addons/mail/static/src/views/web/activity/activity_controller.js

```diff
@@ -39,6 +40,20 @@ export class ActivityController extends Component {
         this.action = useService("action");
         this.activity = useService("mail.activity");
         this.ui = useState(useService("ui"));
+        usePager(() => {
+            const root = this.model.root;
+            const { count, hasLimitedCount, limit, offset } = root;
+            return {
+                offset: offset,
+                limit: limit,
+                total: count,
+                onUpdate: async (params) => {
+                    await this.model.root.load(params);
+                    this.model.fetchActivityData(params);
```

Comment:
nitpicking: the root variable is not very useful as it is used only once and do not improve readability